.gitignore

```
@@ -233,3 +233,6 @@ licenses/LICENSES-ui.txt
 
 # used to checkout target-branch in CI
 /target-airflow
+
+# Dask Executor tests generate this directory
+!/tests/executors/dask-worker-space/
```

```suggestion
/tests/executors/dask-worker-space/
```

According to PR title we need to ignore this directory.

BTW could we control location of this directory. And do we sure that this directory always created here, and do not depend on current working directory?

---

> BTW could we control location of this directory. And do we sure that this directory always created here, and do not depend on current working directory?

Possibly, but this is a historical integration that pretty much no-one here has any experience (or Dask account to test with). I attempted twice to remove it and initiated disussion when it broke our CI (involving writing to DASK discourse) - so far the Dask maintainers managed always to step-in and save the day by fixing the CI issues before we got voting on removal of it (too bad), but I personally treat it as something I would not touch without a stick.